### ISSUE TITLE
datatables - sorting the columns with a key press holds focus

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#3281]
I'm not entirely sure what the behaviour should be, but I'm using this - https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/sortable-table/ - as the reference point.

The issue here is that when you use a screen reader on data-tables (let's take the file browsers' datatable for reference) and sort the column (pressing `enter` while on that table header element), you keep focus on that table header. So using arrow keys (up and down) to navigate to the next element no longer work.

Instead arrow keys start to control the scroll bar, which I believe is incorrect. 

The correct behavior I believe should be that when you press enter to sort a column, the directional keys (up and down) should continue to work and navigate to the next or previous elements.

**Comment by Bubballoo3** [comment]
I am unable to replicate the normal behavior for this using 4.1. That is, the arrow keys never work for table navigation.
